content/authoring/guidelines/kata.md

```diff
@@ -0,0 +1,45 @@
+---
+kind: reference
+sidebar: authoring
+prev: /authoring/guidelines/translation/
+---
+
+# Creating a Kata
+
+This article contains a set of guidelines, which can be used by authors to create good, enjoyable, high quality kata. They were collected to help ensure that kata are of sufficient quality and users' experience will be as good as possible.
+
+
+## Idea
+  
+- **Make sure your kata adds some new value to Codewars.** Avoid creating blatant duplicates, and use the idea which has not yet been widely used to create a kata. Note that with so many existing kata, this can be very challenging to achieve, because it's very dificult to find out if your idea has already been used or not. Remember that you can always ask more experienced users if your idea is worth of creating a new kata, or maybe it has been already exploited sufficiently.
+- **Do not use problems from other sites,** unless it's allowed by that site's licensing policy and/or terms of use. Explicitly give appropriate credit if your kata is based on, or inspired by, someone else's work.
+ 
+
+## Task and Requirements
+ 
+- **Requirements and design should hold to generally accepted and widely recognized guidelines** related to software design, programming, and domain of the kata. Do not require users to do things in a way which is widely recognized as wrong. If any of these guidelines are broken, the description should justify it with a plausible explanation.
+- **Keep kata focused on one task.** Avoid distractions, do not add other, smaller, "side" requirements which do not add too much value to the task itself. 
+- **Make requirements very clear.** If kata is not a puzzle, describe everything what will be tested and leave no surprises. Reason for every failing tests case should be evidently visible in the description of the kata.
```

Comment:
`what will` should be `that will`, `reason for` is missing a `the` before it, and `failing tests` shouldn't be plural because of the `every`